gwpy/cli/qtransform.py

```diff
@@ -149,6 +151,8 @@ def fformat(x):  # float format
                 return '[{0}]'.format(', '.join(map(fformat, x)))
             if isinstance(x, Quantity):
                 x = x.value
+            elif isinstance(x, str):
+                x = float(x)
```

Comment:
if you make the first change, casting the input `args.frange` and `args.qrange` to a `float`, under what circumstances is `x` a `str`?